### ISSUE TITLE
Feature request:  non-modal transients

### ISSUE THREAD

**Issue by davep-github** [magit/transient#35]

**Comment by davep-github** [comment]
Is there anyway to make the transient menus non-modal like they were in popup?  I can't do anything while in the middle of a transient sequence.  I use a single emacs for everything and I can't use it to do anything else while a transient is active.  The old way, using a local map in the popup(now transient), allowed me to do anything else that I needed to do.  I can't think of anything that would be lost by requiring the transient buffer to be active in order to use it.
This isn't the way emacs usually does things.  The only modal things I can think of offhand are mini-buffer related.
I've looked at the code, and it looks like most of the modality is caused by hooking the pre&post command hooks, some mini-buffer hooks and so forth.  Would be be as easy as just not hooking them and making the keymap local, or are there other plumbing or odd code flow issues that could leave things in a bad state?
